website/client-old/js/services/alertServices.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
I'm not thrilled about perpetuating our use of alert windows, but since this will (hopefully) be short-lived code, I'm not going to require that you do anything different.